sync/fromDisk/assessments.js

```diff
@@ -71,6 +71,7 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
         student_group_create: !!assessment.studentGroupCreate,
         student_group_join: !!assessment.studentGroupJoin,
         student_group_leave: !!assessment.studentGroupLeave,
+        question_params: assessment.questionParams || {},
```

Comment:
Can be null in this case.

Reply:
Unsure; do you mean `assessment.questionParams || null`?

Reply:
I mean it can be just `assessment.questionParams`, since if it's null it could be used as is. But I'm ok with the original too.